src/parser/AbstractParser-Commands.cpp

```diff
@@ -876,3 +888,70 @@ bool AbstractParser::evalSpecialCommandMap(StringView args)
     const auto matched = std::vector<StringView>{StringView{s}};
     return rec.callback(matched, args);
 }
+
+void AbstractParser::doCreateInfoMark(const StringView &cmd)
```

Comment:
I would prefer we use the new command syntax over the simple one. An example of this can be seen here: https://github.com/MUME/MMapper/commit/758461088fe885cfea5d85fcc87844957b18f45b